playwright/test/work.test.ts

```diff
@@ -7,11 +7,15 @@ import {
 import { Page } from 'playwright';
 
 const getWhereToFindItAndEncoreLink = async (page: Page) => {
-  const whereToFindIt = await page.$('h2:has-text("Where to find it")');
-  const encoreLink = await page.$('a:has-text("Request item")');
-  const unavailableBanner = await page.$(
-    "[data-test-id='requesting-disabled']"
-  );
+  const whereToFindIt = await page.getByRole('heading', {
+    name: 'Where to find it',
+  });
+
+  const encoreLink = await page.getByRole('link', {
```

Comment:
we were wondering about this name when looking quickly, should it be `anchorLink`?

Reply:
I think the name is `encoreLink` as it is [referencing the name of the library system](https://www.iii.com/products/encore/) that will be logged into. Agree this is quite opaque - how about `loginLink`?

Reply:
Oh wow thanks I had no idea! I think that link is just available when the user is already logged in, so it's not really a login link? (or is it?) `requestLink`?

Reply:
So this raises a good point, the test actually sees this:

<img width="1043" alt="Screenshot 2023-12-19 at 15 21 09" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/953792/4de5b33b-d3ce-493b-86e1-b3bffb167031">

Because it's not signed in and incidentally passes because the "name" is getting lower cased and matches some of the text. I'll update the role name rather than the variable name to reflect what's actually going on here.

Reply:
Oh oops! Something that could also be trialled would be to pass `{exact: true}` as an option
```
page.getByRole('link', {
    name: 'Request items', 
    exact: true
  });
  ```
  
  but yeah in that case I'm a bit confused. It's a link?

Reply:
Ah silly me, it is a link, but I'm guessing hovered on in the screenshot, sorry!